src/AmalgamatorSession.ts

```diff
@@ -637,12 +653,37 @@ export class AmalgamatorSession extends LoggingDebugSession {
         response: DebugProtocol.Response,
         args: any
     ): Promise<void> {
-        // XXX: Which childDap to send a customRequest to? Probably needs domain specific knowledge.
-        const childResponse = await this.childDaps[0].customRequest(
-            command,
-            args
-        );
-        response.body = childResponse.body;
-        this.sendResponse(response);
+        if (command === 'cdt-amalgamator/getChildDapNames') {
+            if (typeof args.child !== 'number') {
```

Comment:
Why does getChildDapNames have a child arg?

Reply:
I deleted it.